addon/ng2/commands/set.ts

```diff
@@ -11,10 +12,38 @@ const SetCommand = Command.extend({
     { name: 'global', type: Boolean, default: false, aliases: ['g'] },
   ],
 
+  asBoolean: function (raw: string): boolean {
+    if (raw == 'true' || raw == '1') {
+      return true;
+    } else if (raw == 'false' || raw == '' || raw == '0') {
+      return false;
+    } else {
+      throw new SilentError(`Invalid boolean value: "${raw}"`);
+    }
+  },
+  asNumber: function (raw: string): number {
+    if (Number.isNaN(+raw)) {
+      throw new SilentError(`Invalid number value: "${raw}"`);
+    }
+    return +raw;
+  },
+
   run: function (commandOptions, rawArgs): Promise<void> {
     return new Promise(resolve => {
-      const config = new CliConfig();
-      config.set(rawArgs[0], rawArgs[1], commandOptions.force);
+      const [jsonPath, rawValue] = rawArgs;
```

Comment:
`rawValue` should just be `value`, no need for `let value = rawValue;` in line 36.